Tools/AP_Bootloader/board_types.txt

```diff
@@ -400,6 +406,8 @@ AP_HW_CUAV-7-NANO                    7000
 # IDs 7100-7109 reserved for VIEWPRO
 AP_HW_VUAV-V7pro                     7100
 
+# IDs 7100-7109 reserved for VIEWPRO
```

Comment:
Is this a typo?

Reply:
I have changed it to 10 ID'S and removed the TYPO.